### PR TITLE
add zeus slots

### PR DESCRIPTION
**Pull request by kil0byt3**
~~i need to figure out why this doesn't work even though it should (?)~~

adds 3 zeus slots with linked gmaster modules so anyone slotted always gets zeus 

Edit: also doubled spectator slots

### CODE REVIEW

**Comment by kil0byt3** [comment]
will fix: #192

**Comment by kil0byt3** [comment]
also this is just blufor maybe zeus slots should not be blufor. idk I'll figure it out

**Comment by AChesheireCat** [comment]
Do you particularly care if the solution is a aesthetically messy one? If not, the easiest way to edit the mission.sqm is actually via the 3den editor. Plop down the units and they'll append themselves to the bottom and update the item counts automatically. I've got a working proof-of-concept if you'd like me to share.

**Comment by kil0byt3** [comment]
That's what I did, I just only used the relevant parts

**Comment by AChesheireCat** [comment]
Try updating line 979 to reflect added items (currently value is 220, should be 225). That seemed to fix the issue in local testing.

Recommend changing zeus unit description on line 45391 to follow lobby screen formatting (description="Zeus 1@BluFor Zeuses";). The line following the @ describes the lobby group name.

**Comment by kil0byt3** [comment]
ty, rf review

**Comment by AChesheireCat** [comment]
The subsequent units in the group do not need the @BluFor Zeuses in their description - the leader of the group is sufficient to name the group on the lobby screen.

ie. Zeus 1 need to be "Zeus 1@BluFor Zeuses" while Zeus 2 just needs to be "Zeus 2", etc. etc.

Line 45391 is missing an @ anyways :P

**Comment by kil0byt3** [comment]
i'll repeat for opf/ind if that lg2u

**Comment by kil0byt3** [comment]
@AChesheireCat can you test this on your end?

**Comment by AChesheireCat** [comment]
> @AChesheireCat can you test this on your end?

Just booted it up in local mp, no errors to be seen. The zeus modules being offset from the characters makes me cringe in the editor, but that's a personal problem lmao. LGTM.

**Comment by kil0byt3** [comment]
Lmfao, listen I am lazy, it works right ;)

File: mission.sqm [hunk 1]
```diff
@@ -45368,5 +45368,950 @@ class Mission
...
+					{
+						position[]={630.7298,5.0014391,834.22571};
+					};
+					side="West";
+					flags=7;
+					class Attributes
+					{
+						rank="COLONEL";
+						name="UNIT_BLU_ZEUS_1";
+						description="Zeus 1@BluFor Zeuses";
```
Comment (PabstMirror) [inline mission.sqm:45391]:
```suggestion
						description="Zeus 1@BluFor Zanes";
```
Plural of Zeus is Zanes

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**